README.md

```diff
@@ -37,6 +43,12 @@ ftl_installer --help
 
 ## Running tests
 
+Run `make install` to install the test dependencies.
+
+```
+make install
+```
+
```

Comment:
@curioswati Instead of running make install as a separate step here, it would probably be better to add a step to the venv target of the Makefile to install the test requirements there.

I would hate fpr someone to accidentally skip the activation of the virtualenv and accidentally install the package and test requirements to their local system.